src/test/ui/issues/issue-60218.stderr

```diff
@@ -1,8 +1,8 @@
 error[E0277]: the trait bound `&u32: Foo` is not satisfied
-  --> $DIR/issue-60218.rs:18:27
+  --> $DIR/issue-60218.rs:18:19
    |
 LL |     trigger_error(vec![], |x: &u32| x)
-   |     -------------         ^^^^^^^^^^^ the trait `Foo` is not implemented for `&u32`
+   |     ------------- ^^^^^^ the trait `Foo` is not implemented for `&u32`
```

Comment:
This one is weird -- the closure in the next arg is actually what causes us to infer that the type of this `vec![]` is `Vec<u32>`, so there's no good way of making this actualy point to the second argument I think :confused:.

Reply:
That's fine. A shame though.